docs/cli/skill_command.md

```
@@ -0,0 +1,129 @@
+# Logout
+datus skill logout --marketplace http://my-town:9000
+```
+
+### List Local Skills
+
+```bash
+datus skill list
+```
+
+Output:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)